### PR TITLE
make Sign Up more obvious

### PR DESCRIPTION
**Pull request by hairedfox**
### Why
- Make the Sign Up more obvious

### What
- Add button Sign Up next to Log In on the Navbar
- Add button Sign Up right next to Log In button in Login Form

### Testing
image

### Pre-Merge Checklist

- [ ] Security & accessibility have been considered
- [ ] Tests have been added, or an explanation has been given why the features cannot be tested
- [ ] Documentation and comments have been added to the codebase where required
- [ ] Entry added to CHANGELOG.md if appropriate
- [ ] Outstanding questions and concerns have been resolved
- [ ] Any next steps have been turned into Issues or Discussions as appropriate

### CODE REVIEW

**Comment by hairedfox** [comment]
#872

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -38,6 +38,7 @@
             </div>
             <div class="" id="login_button">
               <%= f.button :submit, "Log in", class: "button is-primary" %>
+              <%= link_to "Sign Up", new_user_registration_path, class: "button is-success" %>
```
Comment (exbinary) [inline app/views/devise/sessions/new.html.erb:41]: I think this should probably not live inside the `#login_button` div and instead be in a separate sibling div.

File: app/views/devise/shared/_links.html.erb [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <%= link_to "Log in", new_session_path(resource_name) %><br />
 <% end %>
 
-<%- if devise_mapping.registerable? && controller_name != 'registrations' %>
+<%- if devise_mapping.registerable? && !controller_name.in?(['registrations', 'sessions'])  %>
```
Comment (exbinary) [inline app/views/devise/shared/_links.html.erb:5]: TIL about `in?`! 🙏🏾

**Review by exbinary — APPROVED** [review]
@hairedfox , this looks great! 💥 
Just one minor comment.

File: app/views/devise/sessions/new.html.erb (hunk 1 above)
Comment (hairedfox) [reply app/views/devise/sessions/new.html.erb:41]: Thank you for reminding me. I'll fix this one and merge.

**Pull request merged**